### ISSUE TITLE
modernizr path incorrect

### ISSUE THREAD

**Issue by waywardmonkeys** [snide/sphinx_rtd_theme#178]
In ``layout.html``, the path to ``modernizr.min.js`` is given as:

````html
  <script src="_static/js/modernizr.min.js"></script>
````

It should be:

````html
  <script src="{{ pathto('_static/js/modernizr.min.js', 1) }}"></script>
````

**Comment by vkruoso** [comment]
I've encountered this issue myself, and the proposed solution indeed worked. The `linkchecker` utility was showing the errors and now it is passing correctly.

Probably the wrong place to ask, but is this the kind of error that `sphinx-build -b linkcheck` should catch?

**Comment by vkruoso** [comment]
This seems to be left open. It was fixed at #195. Also #213 was about it but was already closed.

Any comments on the ``sphinx-build -b linkcheck`` comment above? This might be a good test to add to the sample docs, and maybe add a CI system like travis to check if no links are broken?

**Comment by agjohnson** [comment]
I believe linkcheck would be of use here, thought I haven't used it myself. It might be a good step for CI if we want to add some test cases for the theme. Feel free to raise linkchecking in a PR if you'd like to tackle that issue.

Closing this as resolved otherwise. I'm putting together a release for 0.1.9 that will include the fix for this issue.
